fire-hose-management-frontend/src/components/Departments/CreateDepartmentOverlay.jsx

```diff
@@ -0,0 +1,206 @@
+import * as i18nIsoCountries from "i18n-iso-countries";
+import React, {useState} from "react";
+import Stack from "@mui/material/Stack";
+import {Button, Checkbox, FormControlLabel, Grid, MenuItem} from "@material-ui/core";
+import {CTextField} from "../shared/input/CTextField";
+import {CSelect} from "../shared/input/CSelect";
+import {ValidatorForm} from "react-material-ui-form-validator";
+import {Overlay} from "../shared/Overlay";
+import {makeStyles} from "@material-ui/core/styles";
+import {useMutation} from "react-query";
+import {postDepartment} from "../../utils/requests/Departments";
+import Notifications from "../shared/Notifications";
+
+const useStyles = makeStyles((theme) => ({
+    checkbox: {
+        display: "table-cell",
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
    display: "table-cell",
```